docs/user/security/reporting.asciidoc

```diff
@@ -32,13 +69,15 @@ POST /_security/user/reporter
 }
 ---------------------------------------------------------------
 
-* If you are using an LDAP or Active Directory realm, you can either assign
+==== If you are using an LDAP or Active Directory realm
+
+You can either assign
 roles on a per user basis, or assign roles to groups of users. By default, role
 mappings are configured in
 {ref}/mapping-roles.html[`config/shield/role_mapping.yml`].
```

Comment:
Note to future me and future you:

https://github.com/elastic/kibana/pull/52559 is introducing a UI to manage role mappings, so when the time comes, we can link there instead of promoting the file-based role mapping mechanism.